catalog.html

```diff
@@ -16,96 +16,119 @@
-                <span>Мотель</span>
-                <span>от 3000 Р.</span>
-                <div class="button-wrapper">
-                  <a href="#">Подробнее</a>
-                  <a href="#">Забронировать</a>
-                </div>
-              </div>
-            </article>
-          </li>
-          <li class="hotel-item">
-            <article class="hotel-card">
-              <img src="img/villas-at-poco-diablo.jpg" alt="Апартаменты в Седоне" width="135" height="90" class="hotel-card-image">
-              <div class="hotel-card-wrapper">
-                <h3 class="hotel-card-title"><a href="#">Villas at Poco Diablo</a></h3>
-                <span>Апартаменты</span>
-                <span>от 2000 Р.</span>
-                <div class="button-wrapper">
-                  <a href="#">Подробнее</a>
-                  <a href="#">Забронировать</a>
-                </div>
-              </div>
-            </article>
+          <li class="sorting-item">
+            <a href="#!" class="sorting-link">
+              по убыванию
+              <svg width="11" height="10" viewBox="0 0 11 10" >
+                <path d="M5.5 10L0 0H11"/>
+                </svg>
+            </a>
           </li>
         </ul>
+        </div>
+        
+         <section class="hotel-card-wrapper">
+          <article class="hotel-card">
+            <img src="img/amara-resort-spa.jpg" alt="фотография гостиницы" class="hotel-image" width="135" height="90" class="hotel-image">
+            <div class="hotel-wrapper">
+              <h3 class="hotel-title"><a href="#" class="hotel-link">Amara Resort & Spa</a></h3>
+              <span class="hotel-subtitle">Гостиница</span>
+              <span class="hotel-subtitle">от 4000р.</span>
+              <div class="button-wrapper">
+                <a href="#" class="hotel-button">Подробнее</a> 
+                <a href="#" class="hotel-button">Забронировать</a>  
+              </div>
+            </div>
+            <div class="rating-wrapper">
+              <div class="star-rating">
+                4 звезды
+              </div>
+              <span class="rating-text">Рейтинг: 8,5</span>
+            </div>
+          </article>
+  
+          <article class="hotel-card">
+            <img src="img/desert-qualin.jpg" alt="фотография мотеля" class="hotel-image" width="135" height="90" class="hotel-image">
+            <div class="hotel-wrapper">
+              <h3 class="hotel-title"><a href="#" class="hotel-link">Desert Quail Inn</a></h3>
+              <span class="hotel-subtitle">Мотель</span>
+              <span class="hotel-subtitle">от 3000р.</span>
+            <div class="button-wrapper">
+              <a href="#" class="hotel-button">Подробнее</a>   
+              <a href="#" class="hotel-button">Забронировать </a>   
+            </div>
+            </div>
+            <div class="rating-wrapper">
+              <div class="star-rating motel">
+                3 звезды
```

Comment:
должен быть класс, обозначающий что стилями нужно вывести именно 3 звезды:
```suggestion
              <div class="star-rating stars-3">
                3 звезды
```